shared/profile/user/weboftrust/container.tsx

```diff
@@ -44,18 +44,21 @@ const Connected = Container.connect(
           : null,
     }
   },
-  (stateProps, dispatchProps) => ({
-    attestation: stateProps.attestation,
-    attestingUser: stateProps.attestingUser,
-    onAccept: dispatchProps._onAccept,
-    onReject: dispatchProps._onReject,
-    reactWaitingKey: wotReactWaitingKey,
-    status: stateProps.status,
-    userIsYou: stateProps.userIsYou,
-    username: stateProps.username,
-    verificationType: stateProps.verificationType,
-    vouchedAt: stateProps.vouchedAt,
-  })
+  (stateProps, dispatchProps) => {
+    const noop = () => null
```

Comment:
Move this to the top of the file. Each time mergeProps gets called (which is lot), you'll be allocating a new `noop` function.

Also, by default functions without a return value in JavaScript return `undefined`.

Reply:
i'm going to be giving some love to accept/reject functions soon. this was kind of a least-necessary-touching approach. but yes on moving noop up. done.